lib/app/connector.rb

```diff
@@ -23,13 +24,12 @@ def start!
         start_polling_jobs
       end
 
-      def create_connector(index_name)
+      def create_connector(index_name, force: false)
         connector_settings = Framework::ConnectorSettings.fetch(App::Config['connector_package_id'])
 
-        if connector_settings.nil?
-          Framework::ElasticConnectorActions.create_connector(index_name)
-
-          connector_settings = Framework::ConnectorSettings.fetch(App::Config['connector_package_id'])
+        if connector_settings.nil? || force
```

Comment:
I'd break the condition in such a way, that if the create_connector is called, but connector is there it should log that the connector has already been created, so skipping it. If `force` flag is passed, it should also note that connector has already been created, but now we're force overwriting it (maybe also providing what's the diff between existing connector setting and what we're writing for it if possible)

Reply:
It's working this way already. Here's the flow:
```
Please select the command:
    start synchronization (sync)
--> register connector with Elasticsearch (register)
    check the status of a third-party service (status)
    end the program (exit)
You already have registered a connector with ID: vbJiyYEB-Xa53KAxjgk5. Registering a new connector will not use the existing one.
Are you sure you want to continue? (y/n)
y
Please enter index name for data ingestion. Use only letters, underscored and dashes.
two
I, [2022-07-05T08:16:00.577987 #6964]  INFO -- : Successfully registered connector two with ID v7JuzYEB-Xa53KAxNAlC
Please store connector ID in config file and restart the program.
```